### PR TITLE
DOC: GH10414 Missing example in NA values in GroupBy

### LINKED ISSUE #10414: Documentation: Example missing in NA values by GroupBy
http://pandas.pydata.org/pandas-docs/stable/missing_data.html#na-values-in-groupby

```
NA groups in GroupBy are automatically excluded. This behavior is consistent with R, for example.
```

I can take this up.

### PR DESCRIPTION
**Pull request by harshnisar**
closes #10414

### CODE REVIEW

**Comment by jreback** [comment]
@harshnisar see the contributing docs [here](http://pandas.pydata.org/pandas-docs/stable/contributing.html)

pls pull current master and rebase. I know its only a trivial change, but this is in conflict as master changed a tiny bit in this section a couple of days ago.

**Comment by harshnisar** [comment]
@jreback

Done :)

File: doc/source/missing_data.rst [hunk 1]
```diff
@@ -176,7 +176,16 @@ NA values in GroupBy
 NA groups in GroupBy are automatically excluded. This behavior is consistent
 with R, for example.
 
+<<<<<<< HEAD
 See the groupby section :ref:`here <groupby.missing>` for more information.
+=======
+.. ipython:: python
```
Comment (jreback) [inline doc/source/missing_data.rst]: you need to remove the conflict lines
further I would move your change above the see further reference

Comment (harshnisar) [inline doc/source/missing_data.rst]: @jreback
Embarrassing. Sorry.

File: doc/source/missing_data.rst [hunk 2]
```diff
@@ -174,9 +174,15 @@ NA values in GroupBy
...
-with R, for example.
+with R, for example:
+
+.. ipython:: python
+
+    df
+    df.groupby('one').mean()
 
 See the groupby section :ref:`here <groupby.missing>` for more information.
+=======
```
Comment (shoyer) [inline doc/source/missing_data.rst]: I think this snuck in during a rebase/merge?

Comment (harshnisar) [inline doc/source/missing_data.rst]: @shoyer In my defence, it really did fit well. Changed :+1: Embarrassed.

**Comment by jorisvandenbossche** [comment]
Looks good now! @harshnisar Last question: can you squash the three commits into one? (see end of this section: http://pandas.pydata.org/pandas-docs/stable/contributing.html#committing-your-code)

**Comment by harshnisar** [comment]
@jorisvandenbossche  I know by now, this pull request is more of an effort sink for you than a contribution by me. Won't happen from next pulls.

**Comment by jorisvandenbossche** [comment]
@harshnisar no, no problem! And thanks!

**Pull request merged**